core/src/Kokkos_Macros.hpp

```diff
@@ -254,33 +186,6 @@
 #define KOKKOS_COMPILER_MSVC _MSC_VER
 #endif
 
-//#endif // #if !defined( __CUDA_ARCH__ )
-//----------------------------------------------------------------------------
-// Language info: C++, CUDA, OPENMP
-
-#if defined(KOKKOS_ENABLE_CUDA)
-// Compiling Cuda code to 'ptx'
-
-#define KOKKOS_IMPL_FORCEINLINE_FUNCTION __device__ __host__ __forceinline__
-#define KOKKOS_IMPL_FORCEINLINE __forceinline__
-#define KOKKOS_IMPL_INLINE_FUNCTION __device__ __host__ inline
-#define KOKKOS_IMPL_FUNCTION __device__ __host__
-#define KOKKOS_IMPL_HOST_FUNCTION __host__
-#define KOKKOS_IMPL_DEVICE_FUNCTION __device__
-#if defined(KOKKOS_COMPILER_NVCC)
-#define KOKKOS_INLINE_FUNCTION_DELETED inline
-#else
-#define KOKKOS_INLINE_FUNCTION_DELETED __device__ __host__ inline
-#endif
-#if (CUDA_VERSION < 10000)
-#define KOKKOS_DEFAULTED_FUNCTION __host__ __device__ inline
-#else
-#define KOKKOS_DEFAULTED_FUNCTION inline
-#endif
-#define KOKKOS_IMPL_HOST_FUNCTION __host__
-#define KOKKOS_IMPL_DEVICE_FUNCTION __device__
-#endif
```

Comment:
The subsequent lines for the HIP backend have also been moved to `src/setup/Kokkos_Setup_HIP.hpp`.

Reply:
Addressed in f63ecf1